cmd/keytransparency-client/cmd/root.go

```diff
@@ -76,11 +76,11 @@ func init() {
 	cobra.OnInitialize(initConfig)
 	RootCmd.PersistentFlags().StringVar(&cfgFile, "config", "", "config file (default is $HOME/.keytransparency.yaml)")
 
-	RootCmd.PersistentFlags().String("domain", "google.com", "Domain within the KT server")
-	RootCmd.PersistentFlags().String("kt-url", "35.184.134.53:8080", "URL of Key Transparency server")
+	RootCmd.PersistentFlags().String("domain", "default", "Domain within the KT server")
+	RootCmd.PersistentFlags().String("kt-url", "35.225.194.168:8080", "URL of Key Transparency server")
```

Comment:
Does it make sense to hardcode an IP address?

Reply:
It helps everything "just" work, for now.

Reply:
Default addresses for load tests can be risky. I've seen outages caused by this, though on GCE it's probably not as dangerous. e.g. if you'd put some internal network address and someone downloads it who happens to have some server running on that address.